mlocal/scripts/find_program

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Is there a good reason to use this over standard programs like `which` and `command -v`?

Reply:
When I was doing this I had something else in mind, and I ended up backtracking. So, right, the script is equivalent to command -v (except that command -v considers aliases, too, but I'll ignore that).